search-endpoint.md

```diff
@@ -0,0 +1,397 @@
+# Discovery Search Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Search Endpoint
+**/api/discover/search**   
+
+This endpoint provides the functionality of the Discovery search screen: http://demo.dspace.org/xmlui/discover.
```

Comment:
Let's not link to XMLUI here please ;)  Maybe just link to the Discovery docs for now? https://wiki.duraspace.org/display/DSDOC6x/Discovery